CHANGELOG.rst

```diff
@@ -1,5 +1,8 @@
 Changelog for Mantarray Frontend Components
 ===========================================
+1.2.6 (2022-11-10)
```

Comment:
There is already an unreleased version 1.2.5 below on line  7, you can add your changes to that version

Reply:
The FE  for mantarray doesn't work the same as the Pulse3D repo where each PR needs a new version and thats because the publish workflow gets run on merge for Pulse3D, whereas we have to manually run a publish workflow for this repo so we can have multiple PRs merged under the same version until we are ready to release it.

Reply:
You don't actually need to bump the version in each PR in the pulse3D repo either. If you don't bump the version, the publish will just fail. We just happen to bump the pulse3d version in almost all PRs because we typically need to immediately bring the pulse3d changes into the k8s repo and test them out. If there were an easier way to do that (i.e. install from branch which unfortunately is very difficult for pulse3d for various reasons), then we would probably only bump the version and publish right before we merge the RC branch. There's also no reason we couldn't have the publish workflow run automatically on PR merge in this repo too, it just hasn't been set up yet